app/templates/components/preprint-form-header.hbs

```diff
@@ -26,10 +26,10 @@
 {{#if (and (not isOpen) (eq name 'Upload'))}}
     <div class="m-t-md preprint-header-preview">
         {{#if preprintNode}}
-            <p><em class="m-r-md"> {{t "components.preprint-form-header.location"}}: </em> {{get-ancestor-descriptor preprintNode}} {{preprintNode.title}}</p>
+            <p><em class="m-r-md"> {{t "components.preprint-form-header.location"}}: </em> {{get-ancestor-descriptor preprintNode}} {{safe-markup preprintNode.title}}</p>
```

Comment:
Per previous discussion, remember that this PR **cannot** be merged so long as this line is present (see also line 32 below, etc). 

The `safe-markup` helper should not be used with user-entered text.